webrender_api/src/image.rs

```diff
@@ -110,11 +112,26 @@ impl ImageDescriptor {
 
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub enum ImageData {
-    Raw(Arc<Vec<u8>>),
-    Blob(BlobImageData),
+    Raw(#[serde(with = "serde_image_data_raw")] Arc<Vec<u8>>),
+    Blob(#[serde(with = "serde_bytes")] BlobImageData),
     External(ExternalImageData),
 }
 
+mod serde_image_data_raw {
```

Comment:
This is necessary because we are dealing with an `Arc`. However, maybe it would be better to patch `serde_bytes` to support this?

Reply:
Cc @dtolnay

Reply:
Could you propose a more general signature for serde_bytes that could work here?

Reply:
Perhaps not, though I haven't looked into it very closely. Assuming a more general signature is not workable, maybe `serde_bytes` could also ship `serde_bytes_rc`/`serde_bytes_arc` modules though? (Essentially what I've done here, but inside the `serde_bytes` crate.)